README.md

```diff
@@ -47,26 +47,70 @@ it('should demonstrate this matcher`s usage', async () => {
 
 ```javascript
 const { axe, toHaveNoViolations } = require('jest-axe')
+const App = require('./app)
 
 expect.extend(toHaveNoViolations)
 
 const React = require('react')
-const ReactDOMServer = require('react-dom/server')
 
 it('should demonstrate this matcher`s usage with react', async () => {
-  const html = ReactDOMServer.renderToString(
-    <img src='#' />
-  )
+  const results = await axe(<App/>)
+  expect(results).toHaveNoViolations()
+})
+```
 
-  const results = await axe(html)
+### With React Testing Library

```

Comment:
Where possible I've tried to keep the test fixtures as close to the README snippets as possible.

Is it possible to keep them the same, the README examples seem simpler than what is being tested, are they correct?

Reply:
I've updated all of the tests and README to match as closely as possible